consensus/mint-client/src/config.rs

```diff
@@ -20,21 +163,30 @@ pub enum Commands {
         #[clap(long, env = "MC_CONSENSUS_URI")]
         node: ConsensusClientUri,
 
-        /// The key to sign the transaction with.
-        #[clap(long, parse(try_from_str = load_key_from_pem), env = "MC_MINTING_SIGNING_KEY")]
-        signing_key: Ed25519Private,
+        #[clap(flatten)]
+        params: MintConfigTxParams,
+    },
+
+    // Generate a MintConfigTx and write it to a JSON file.
+    GenerateMintConfigTx {
+        /// Filename to write the mint configuration to.
+        #[clap(long, env = "MC_MINTING_OUT_FILE")]
+        out: PathBuf,
 
-        /// The token id we are minting.
-        #[clap(long, env = "MC_MINTING_TOKEN_ID")]
-        token_id: u32,
+        #[clap(flatten)]
+        params: MintConfigTxParams,
+    },
 
-        /// Tombstone block.
-        #[clap(long, env = "MC_MINTING_TOMBSTONE")]
-        tombstone: Option<u64>,
+    // Submit json-encoded MintConfigTx(s). If multiple transactions are provided, signatures will
+    // be merged.
+    SubmitMintConfigTx {
+        /// URI of consensus node to connect to.
+        #[clap(long, env = "MC_CONSENSUS_URI")]
+        node: ConsensusClientUri,
 
-        /// Nonce.
-        #[clap(long, parse(try_from_str = FromHex::from_hex), env = "MC_MINTING_NONCE")]
-        nonce: Option<[u8; NONCE_LENGTH]>,
+        /// Filename to read the mint configuration from.
```

Comment:
Seems useful to specify the format
```suggestion
        /// Paths for the JSON-formatted mint configuration files.
```